app/register/page.tsx

```diff
@@ -112,15 +127,19 @@ const Register: React.FC = () => {
               marginTop: "40px",
               fontWeight: "bold",
               color: '#11142D'
-            }}>
+            }}
```

Comment:
Can you only add the lines where the changes are required? Why does this formatting changes getting pushed?